### PR TITLE
deactivate groups

### PR DESCRIPTION
**Pull request by wbooth**

### CODE REVIEW

**Comment by wbooth** [comment]
requires https://github.com/mitre/caldera/pull/235

File: app/chain_api.py [hunk 1]
```diff
@@ -43,7 +43,10 @@ def __init__(self, services):
                 core_agent=lambda d: self.data_svc.explode_agents(criteria=d),
                 core_group=lambda d: self.data_svc.explode_groups(criteria=d),
                 core_result=lambda d: self.data_svc.explode_results(criteria=d),
-            )
+            ),
+            DELETE=dict(
+                core_group=lambda d: self.data_svc.deactivate_group(**d),
```
Comment (privateducky) [inline app/chain_api.py]: when you make this change in the future (linked to the change in the core code) make sure you "link" the two MRs.

**Comment by privateducky** [comment]
this does not seem to work

**Comment by wbooth** [comment]
@privateducky please use in coordination with https://github.com/mitre/caldera/pull/244

**Pull request closed without merging**